### ISSUE TITLE
read secondary structure from file

### ISSUE THREAD

**Issue by csbrasnett** [marrink-lab/vermouth-martinize#613]
Rather than pasting a long string, it might be nice if there was an option to read the secondary structure from some kind of text file. This is slightly inspired by [my PR](https://github.com/Martini-Force-Field-Initiative/Martini-Force-Field-Initiative.github.io/pull/40) and [associated discussion](https://github.com/orgs/Martini-Force-Field-Initiative/discussions/5) for the website and tutorial, because I think the current explanation is confusing to people not familiar with martinize2.

Maybe this is more of a documentation problem. I started having a go at [writing a wiki](https://github.com/csbrasnett/vermouth-martinize/wiki) over the summer, but that still needs a bit more work. Thoughts welcome

**Comment by pckroon** [comment]
Alright, that sounds like a useful feature. Do you mean putting the SS string in a file, or the output from dssp? Both sound reasonable.

What do you think is better, a github wiki, or readthedocs documentation (https://vermouth-martinize.readthedocs.io/en/latest/?badge=latest), or do they serve different purposes (which?)? @fgrunewald I'm also interested in your take on this.

@csbrasnett I'm *terrible* with documentation, so any help is always much appreciated.

**Comment by fgrunewald** [comment]
hey guys, isn't much of the confusion because martinize(v1) could read a ss-string from file (i.e. not the dssp output but the already done string). So this is in favor. On the other hand why would you want to do this in the first place? I can't really think of a use case.

**Comment by csbrasnett** [comment]
I agree it's not urgent, but it'd be nicer than pasting a massive string in the terminal as we currently describe. That's about as much use case as I can imagine, particularly now there's the mdtraj functionality. I didn't know that martinize1 had that functionality, but maybe that's part of the user-based confusion in that case.

@pckroon my intuition is that a readthedocs is preferable as a searchable page for people to find separately from the repository. For an 'intro to martinize2' purpose I don't think they serve different purposes. If you just want to use the tool as is, it should be fine, and we can always tell people to open an issue from there if they're struggling?

**Comment by pckroon** [comment]
I scrolled through the wiki you wrote. I'd love to have it incorporated into the regular documentation ;)
I'd suggest to create a super simple wiki that simply says: "Go read the documentation over there". And yes, people who have issues can always open a discussion thread (rather than an issue IMO)
